x-pack/plugins/apm/server/lib/transactions/get_transaction/index.ts

```diff
@@ -60,3 +60,24 @@ export async function getTransaction(
   const resp = await client<Transaction>('search', params);
   return idx(resp, _ => _.hits.hits[0]._source);
 }
+
+export async function getTransactionWithErrorCount(
+  transactionId: string,
+  traceId: string,
+  setup: Setup
+): Promise<TransactionWithErrorCountAPIResponse> {
+  const transaction = await getTransaction(transactionId, traceId, setup);
+  const errorCount = transaction
+    ? await getErrorCount({
+        serviceName: transaction.service.name,
+        transactionId: transaction.transaction.id,
+        traceId,
+        setup
+      })
+    : 0;
```

Comment:
Sorry for keep going over this but now I understand the issue: `getErrorCount` doesn't need `serviceName`. The combination of `transactionId` and `traceId` makes it globally unique, just like with `getTransaction`.

```js
  const errorCount = await getErrorCount({
    transactionId,
    traceId,
    setup
  });
```